2012/07/16 04:16:53 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in C:\yii\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\yii\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\yii\framework\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 C:\yii\framework\base\CModule.php(388): CDbConnection->init()
#3 C:\yii\framework\base\CModule.php(104): CModule->getComponent('db')
#4 C:\yii\framework\gii\generators\crud\CrudCode.php(44): CModule->__get('db')
#5 C:\yii\framework\web\CFormModel.php(40): CrudCode->init()
#6 C:\yii\framework\gii\CCodeGenerator.php(152): CFormModel->__construct()
#7 C:\yii\framework\gii\CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 C:\yii\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 C:\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 C:\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 C:\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(276): CController->run('')
#13 C:\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('gii/crud')
#14 C:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/sysadd1_midterms/ourspace/index.php?r=gii/crud
HTTP_REFERER=http://localhost:81/sysadd1_midterms/ourspace/index.php?r=gii/controller
---
2012/07/16 04:17:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in C:\yii\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\yii\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\yii\framework\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 C:\yii\framework\base\CModule.php(388): CDbConnection->init()
#3 C:\yii\framework\base\CModule.php(104): CModule->getComponent('db')
#4 C:\yii\framework\gii\generators\model\ModelCode.php(54): CModule->__get('db')
#5 C:\yii\framework\web\CFormModel.php(40): ModelCode->init()
#6 C:\yii\framework\gii\CCodeGenerator.php(152): CFormModel->__construct()
#7 C:\yii\framework\gii\CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 C:\yii\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 C:\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 C:\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 C:\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(276): CController->run('')
#13 C:\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 C:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/sysadd1_midterms/ourspace/index.php?r=gii/model
HTTP_REFERER=http://localhost:81/sysadd1_midterms/ourspace/index.php?r=gii/controller
---
2012/07/16 04:18:40 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in C:\yii\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\yii\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\yii\framework\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 C:\yii\framework\base\CModule.php(388): CDbConnection->init()
#3 C:\yii\framework\base\CModule.php(104): CModule->getComponent('db')
#4 C:\yii\framework\gii\generators\model\ModelCode.php(54): CModule->__get('db')
#5 C:\yii\framework\web\CFormModel.php(40): ModelCode->init()
#6 C:\yii\framework\gii\CCodeGenerator.php(152): CFormModel->__construct()
#7 C:\yii\framework\gii\CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 C:\yii\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 C:\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 C:\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 C:\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(276): CController->run('')
#13 C:\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 C:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/sysadd1_midterms/ourspace/index.php?r=gii/model
HTTP_REFERER=http://localhost:81/sysadd1_midterms/ourspace/index.php?r=gii/controller
---
2012/07/16 04:21:05 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in C:\yii\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\yii\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\yii\framework\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 C:\yii\framework\base\CModule.php(388): CDbConnection->init()
#3 C:\yii\framework\base\CModule.php(104): CModule->getComponent('db')
#4 C:\yii\framework\gii\generators\model\ModelCode.php(54): CModule->__get('db')
#5 C:\yii\framework\web\CFormModel.php(40): ModelCode->init()
#6 C:\yii\framework\gii\CCodeGenerator.php(152): CFormModel->__construct()
#7 C:\yii\framework\gii\CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 C:\yii\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 C:\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 C:\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 C:\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(276): CController->run('')
#13 C:\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 C:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/sysadd1_midterms/ourspace/index.php?r=gii/model
HTTP_REFERER=http://localhost:81/sysadd1_midterms/ourspace/index.php?r=gii/default/index
---
2012/07/16 04:21:55 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in C:\yii\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\yii\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\yii\framework\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 C:\yii\framework\base\CModule.php(388): CDbConnection->init()
#3 C:\yii\framework\base\CModule.php(104): CModule->getComponent('db')
#4 C:\yii\framework\gii\generators\model\ModelCode.php(54): CModule->__get('db')
#5 C:\yii\framework\web\CFormModel.php(40): ModelCode->init()
#6 C:\yii\framework\gii\CCodeGenerator.php(152): CFormModel->__construct()
#7 C:\yii\framework\gii\CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 C:\yii\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 C:\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 C:\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 C:\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(276): CController->run('')
#13 C:\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 C:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/sysadd1_midterms/ourspace/index.php?r=gii/model
HTTP_REFERER=http://localhost:81/sysadd1_midterms/ourspace/index.php?r=gii/controller
---
2012/07/16 04:22:44 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in C:\yii\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\yii\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\yii\framework\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 C:\yii\framework\base\CModule.php(388): CDbConnection->init()
#3 C:\yii\framework\base\CModule.php(104): CModule->getComponent('db')
#4 C:\yii\framework\gii\generators\model\ModelCode.php(54): CModule->__get('db')
#5 C:\yii\framework\web\CFormModel.php(40): ModelCode->init()
#6 C:\yii\framework\gii\CCodeGenerator.php(152): CFormModel->__construct()
#7 C:\yii\framework\gii\CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 C:\yii\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 C:\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 C:\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 C:\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(276): CController->run('')
#13 C:\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 C:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/sysadd1_midterms/ourspace/index.php?r=gii/model
HTTP_REFERER=http://localhost:81/sysadd1_midterms/ourspace/index.php?r=gii/controller
---
2012/07/16 04:23:05 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in C:\yii\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\yii\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\yii\framework\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 C:\yii\framework\base\CModule.php(388): CDbConnection->init()
#3 C:\yii\framework\base\CModule.php(104): CModule->getComponent('db')
#4 C:\yii\framework\gii\generators\model\ModelCode.php(54): CModule->__get('db')
#5 C:\yii\framework\web\CFormModel.php(40): ModelCode->init()
#6 C:\yii\framework\gii\CCodeGenerator.php(152): CFormModel->__construct()
#7 C:\yii\framework\gii\CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 C:\yii\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 C:\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 C:\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 C:\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(276): CController->run('')
#13 C:\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 C:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/sysadd1_midterms/ourspace/index.php?r=gii/model
HTTP_REFERER=http://localhost:81/sysadd1_midterms/ourspace/index.php?r=gii/controller
---
2012/07/16 04:23:48 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'sysadd1_midterm'' in C:\yii\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\yii\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\yii\framework\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 C:\yii\framework\base\CModule.php(388): CDbConnection->init()
#3 C:\yii\framework\base\CModule.php(104): CModule->getComponent('db')
#4 C:\yii\framework\gii\generators\model\ModelCode.php(54): CModule->__get('db')
#5 C:\yii\framework\web\CFormModel.php(40): ModelCode->init()
#6 C:\yii\framework\gii\CCodeGenerator.php(152): CFormModel->__construct()
#7 C:\yii\framework\gii\CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 C:\yii\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 C:\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 C:\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 C:\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\yii\framework\web\CWebApplication.php(276): CController->run('')
#13 C:\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 C:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/sysadd1_midterms/ourspace/index.php?r=gii/model
HTTP_REFERER=http://localhost:81/sysadd1_midterms/ourspace/index.php?r=gii/controller
---
2012/07/16 04:27:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "myaddress".' in C:\yii\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('myaddress')
#1 C:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/sysadd1_midterms/ourspace/index.php?r=myaddress
---
2012/07/16 05:04:30 [warning] [application] Failed to set unsafe attribute "home_address" of "Myaddress".
in C:\xampp\htdocs\sysadd1_midterms\ourspace\protected\controllers\MyaddressController.php (71)
in C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php (13)
2012/07/16 05:04:30 [warning] [application] Failed to set unsafe attribute "landline" of "Myaddress".
in C:\xampp\htdocs\sysadd1_midterms\ourspace\protected\controllers\MyaddressController.php (71)
in C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php (13)
2012/07/16 05:04:30 [warning] [application] Failed to set unsafe attribute "cellphone" of "Myaddress".
in C:\xampp\htdocs\sysadd1_midterms\ourspace\protected\controllers\MyaddressController.php (71)
in C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php (13)
2012/07/16 05:07:21 [warning] [application] Failed to set unsafe attribute "home_address" of "Myaddress".
in C:\xampp\htdocs\sysadd1_midterms\ourspace\protected\controllers\MyaddressController.php (95)
in C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php (13)
2012/07/16 05:07:21 [warning] [application] Failed to set unsafe attribute "landline" of "Myaddress".
in C:\xampp\htdocs\sysadd1_midterms\ourspace\protected\controllers\MyaddressController.php (95)
in C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php (13)
2012/07/16 05:07:21 [warning] [application] Failed to set unsafe attribute "cellphone" of "Myaddress".
in C:\xampp\htdocs\sysadd1_midterms\ourspace\protected\controllers\MyaddressController.php (95)
in C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php (13)
2012/07/16 05:08:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "myadress/view".' in C:\yii\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('myadress/view')
#1 C:\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/sysadd1_midterms/ourspace/index.php?r=myadress/view&id=1
---
2012/07/16 05:24:40 [warning] [application] Failed to set unsafe attribute "home_address" of "Myaddress".
in C:\xampp\htdocs\sysadd1_midterms\ourspace\protected\controllers\MyaddressController.php (71)
in C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php (13)
2012/07/16 05:24:40 [warning] [application] Failed to set unsafe attribute "landline" of "Myaddress".
in C:\xampp\htdocs\sysadd1_midterms\ourspace\protected\controllers\MyaddressController.php (71)
in C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php (13)
2012/07/16 05:24:40 [warning] [application] Failed to set unsafe attribute "cellphone" of "Myaddress".
in C:\xampp\htdocs\sysadd1_midterms\ourspace\protected\controllers\MyaddressController.php (71)
in C:\xampp\htdocs\sysadd1_midterms\ourspace\index.php (13)
